Makefile.pre.in

```diff
@@ -1179,7 +1179,7 @@ regen-importlib: regen-frozen
 # Global objects
 
 .PHONY: regen-global-objects
-regen-global-objects: regen-deepfreeze $(srcdir)/Tools/scripts/generate_global_objects.py
```

Comment:
It looks like this was intentionally added in 21099fc064c61d59c936a2f6a0db3e07cd5c8de5, so the `regen-deepfreeze` should stay.

Reply:
However, this dependency causes the `regen-global-objects` to be executed with the compile task first, and it does not compile successfully at this point because of the new symbols(_Py_ID(__orig_class__)) added.